### ISSUE TITLE
Stripper Models Conflicting with LS Map Updates

### ISSUE THREAD

**Issue by larrybrains** [SirPlease/L4D2-Competitive-Rework#172]
There are some ladders and new props that were added with the Last Stand update that conflict with some of the stripper addons in the following maps (see screenshot links for reference).

**c1m2_streets:** 

New blocks added on overpass above dumpster area conflict with stripper blocks: (2 beneath crosshair in screenshot and two more farther down the railing)
https://steamuserimages-a.akamaihd.net/ugc/1664602411081791040/3281D6EA5A2E78DC08CCC5680FF88854774DAA50/

New ladder added at one way drop conflicts with some of the props added via stripper: 
https://steamuserimages-a.akamaihd.net/ugc/1664602411081790522/A67B8752DF5F8C54131ED931FC550E53BC22FCE0/

**c2m4_barns**

New bumper cars added to saferoom in update conflict with the props added via stripper: (image shows stripper props removed)
https://steamuserimages-a.akamaihd.net/ugc/1664602411081789536/D528EF6F3499F6E8640B113BCA76E466730CFBE7/

**c5m3_cemetary**

New ladders on both sides of fence before the sewer drop conflict with props added via stripper. This one is actually a big one as the models combined with the ladder make it difficult to get over the fence, which is especially frustrating as a tank.

https://steamuserimages-a.akamaihd.net/ugc/1664602411081852465/8E3A92A1304D8C30090ED8D2064AAC3E9C4E7BEB/
https://steamuserimages-a.akamaihd.net/ugc/1664602411081852128/44ABCC1FE56DECEC944AED54622FBD8BC689892F/

Also, there are probably more, these are just the ones I have found so far.

**Comment by SirPlease** [comment]
Parish 2 has a lot of them atm, they'll be ironed out soon.
c2m4 already has a fix for it locally, but not pushed to the Github yet.

I appreciate the reports, will look at the other two and get to Parish.

**Comment by larrybrains** [comment]
Not sure if you already have local fixes in place, but here is what I did for my server on a couple of these: (unfortunately I can't just quote line numbers since my larrymod stripper files differ from yours)

**c1m2_streets: - new overpass blocks duplicated fix**

in cfg/stripper/modname/maps/c1m2_streets.cfg remove concrete_block001_128_reference.mdl props with origins:

```
"origin" "-2296 2341 291"
"origin" "-2816 2340 291"
"origin" "-2426 2341 291"
```

**c5m3_cemetary** - new ladders over fence. 

For this one, I wanted to keep some of the props against the fence along side the new infected ladders because they add a spawn point and I think it looks good with the junk there, so I removed the barrels on the one side, and rearranged the dressers and sign on the other. Here is how the final product looks: https://steamuserimages-a.akamaihd.net/ugc/1664602678564813552/7958F86066C3E7C7B076B790E0CF39BA11475D99/?imw=1024&imh=576&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=true

1. In cfg/stripper/modname/maps/c5m3_cemtary.cfg remove oil drum props with origins:

```
"origin" "3815.53 1620 0.113297"
"origin" "3820.49 1637.3 48.1133"
"origin" "3826 1656.53 0.113297"

```
2. on military_sign02_2.mdl prop_dynamic with "origin" "3801 1675 -10.8867" change "origin" to:

`"origin" "3776 1502 -10.8867"`

3. on dresser_tall.mdl prop_dynamic with "origin" "3779 1628 0.113304" change "model" & "origin" to:

```
"model" "models/props_urban/oil_drum001.mdl"
"origin" "3776 1617 0.113304"
```

4. on dresser_short.mdl with "origin" "3768 1558 0.113304" change model to:

`"model" "models/props_interiors/dresser_tall.mdl"`
